.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
To be "precise", it's a combination of specifying `dist: precise` and plain `mysql` (no version) which by default installs `mysql v5.5`... https://github.com/inveniosoftware/invenio-access/blob/master/.travis.yml#L26